pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -2052,6 +2057,147 @@ into_table_name:
         $$ = $1
     }
 
+export_data_stmt:
+    select_no_parens export_data_param
+    {
+        $$.(*tree.Select).Ep = $2
+    }
+|   select_with_parens export_data_param
+    {
+        $$ = &tree.Select{Select: $1, Ep: $2}
+    }
+
+export_data_param:
+    INTO OUTFILE STRING export_fields export_lines_opt header_opt max_file_size_opt force_quote_opt
+    {
+        $$ = &tree.ExportParam{
+            Outfile:    true,
+            FilePath :  $3,
+            Fields:     $4,
+            Lines:      $5,
+            Header:     $6,
+            MaxFileSize:uint64($7),
+            ForceQuote: $8,
+        }
+    }
+
+export_fields:
+    {
+        $$ = &tree.Fields{
+            Terminated: ",",
+            EnclosedBy: '"',
+        }
+    }
+|   TERMINATED BY STRING
+    {
+        $$ = &tree.Fields{
+            Terminated: $3,
+            EnclosedBy: '"',
+        }
+    }
+|   TERMINATED BY STRING ENCLOSED BY field_terminator
+    {
+        str := $6
+        if str != "\\" && len(str) > 1 {
+            yylex.Error("error field terminator")
+            return 1
+        }
+        var b byte
+        if len(str) != 0 {
+           b = byte(str[0])
+        } else {
+           b = 0
+        }
+        $$ = &tree.Fields{
+            Terminated: $3,
+            EnclosedBy: b,
+        }
+    }
+|   ENCLOSED BY field_terminator
+    {
+        str := $3
+        if str != "\\" && len(str) > 1 {
+            yylex.Error("error field terminator")
+            return 1
+        }
+        var b byte
+        if len(str) != 0 {
+           b = byte(str[0])
+        } else {
+           b = 0
+        }
+        $$ = &tree.Fields{
+            Terminated: ",",
+            EnclosedBy: b,
+        }
+    }
+
+export_lines_opt:
+    {
+        $$ = &tree.Lines{
+            TerminatedBy: "\n",
+        }
+    }
+|   LINES lines_terminated_opt
+    {
+        $$ = &tree.Lines{
+            TerminatedBy: $2,
+        }
+    }
+
+header_opt:
+    {
+        $$ = true
+    }
+|   HEADER STRING
+    {
+        str := strings.ToLower($2)
+        if str == "true" {
+            $$ = true
+        } else if str == "false" {
+            $$ = false
+        } else {
+            yylex.Error("error header flag")
+            return 1
+        }
+    }
+
+max_file_size_opt:
+    {
+        $$ = 0
+    }
+|   MAX_FILE_SIZE STRING
```

Comment:
You can add an integer instead of adding a string to convert it to an integer after MAX_FILE_SIZE